suites/pacific/rgw/tier-1-upgrade_4x_to_5x_latest.yaml

```diff
@@ -0,0 +1,412 @@
+#############################################################################
```

Comment:
Could we rename the file to `tier-1_rgw_test-upgrade-4-to-latest.yaml`?

Reply:
Done. Renamed it to tier-3_rgw_test-upgrade-4-to-latest.yaml

Reply:
Renamed it to tier-3_rgw_test-upgrade-4-to-latest.yaml